config/locales/hi.yml

```diff
@@ -0,0 +1,41 @@
+
+hi:
+  errors:
+    messages:
+      taken_in_past: पहले लिया गया 
+      equal_to_current_password: इससे अलग पासवर्ड होना चाहिए 
+      password_complexity:
+        digit:
+          one: एक अंक होना चाहिए 
+          other: कम से कम %{count} अंक होने चाहिए  
+        lower:
+          one: एक लोअर-केस अक्षर होना चाहिए 
+          other: कम से कम %{count} अक्षर होने चाहिए  
+        symbol:
+          one: एक प्रतीक चिन्ह होना चाहिए 
```

Comment:
<img width="1283" alt="Screen Shot 2020-10-08 at 10 52 39 AM" src="https://user-images.githubusercontent.com/1635820/95475606-6321e280-0954-11eb-814c-a3cff02c0307.png">

Don't think this is quite right. The English version is `must contain at least one punctuation mark or symbol`.

Reply:
resolved - https://www.shabdkosh.com/search-dictionary?lc=hi&sl=en&tl=hi&e=symbol